src/main/webapp/app/delete-dialog/delete-dialog.module.ts

```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { JhiLanguageService } from 'ng-jhipster';
+import { JhiLanguageHelper } from 'app/core';
+
+import { ArtemisSharedModule } from 'app/shared';
+import { DeleteDialogComponent } from 'app/delete-dialog/delete-dialog.component';
+import { DeleteDialogService } from 'app/delete-dialog/delete-dialog-service';
+
+@NgModule({
+    imports: [ArtemisSharedModule],
+    declarations: [DeleteDialogComponent],
+    providers: [DeleteDialogService, { provide: JhiLanguageService, useClass: JhiLanguageService }],
+})
+export class ArtemisDeleteDialogModule {
```

Comment:
Should this be part of the `ArtemisSharedModule`?

Reply:
I think this is a very good point - we probably want to enforce this dialog to be used as much as possible.

Reply:
Moved it to `ArtemisSharedModule`